.github/workflows/checks.yml

```
@@ -549,7 +549,7 @@ jobs:
         uses: actions/cache@v4
         with:
           path: /tmp/docker
-          key: ${{ runner.os }}-docker-${{ env.TEST_BITCOIN_DOCKER }}
+          key: ${{ runner.os }}-docker-${{ env.BITCOIN_DOCKER_IMAGE }}
```

what's the reason behind this change?

---

docker-setup step was silently failing, related commit: https://github.com/chainwayxyz/citrea/pull/3075/commits/ba438f0ee37b0f3a2fb42220474b876de0c75713

---

Wasn't pulling the correct bitcoin docker image